generate-proxies.json

```diff
@@ -0,0 +1,23 @@
+{
+    "manifest": "node_modules/@tylertech/forge/custom-elements.json",
+    "importPath": "@tylertech/forge",
+    "exclude": "",
+    "outDir": "projects/forge-angular/src/lib",
+    "outDirExcludePrefix": "forge-",
+    "modulePrefix": "Forge",
+    "useDefineFunction": true,
+    "componentDependencies": {
+        "forge-accordion": ["forge-expansion-panel"],
+        "forge-button-toggle-group": ["forge-button-toggle"],
+        "forge-chip-field": ["forge-chip"],
+        "forge-chip-set": ["forge-chip"],
+        "forge-expansion-panel": ["forge-open-icon"],
+        "forge-icon-button": ["forge-icon"],
+        "forge-list": ["forge-list-item"],
+        "forge-select": ["forge-option"],
+        "forge-split-view": ["forge-split-view-panel"],
+        "forge-stepper": ["forge-step"],
+        "forge-tab-bar": ["forge-tab"],
+        "forge-view-switcher": ["forge-view"]
+    }
```

Comment:
This is a great convenience feature!

Just a comment for the future, but I wonder if we could use the custom elements manifest to define these relationships and consume that here?